Documentation/DocuBlocks/Rest/Administration/get_admin_cluster_rebalance.md

```diff
@@ -4,20 +4,36 @@
 @RESTHEADER{GET /_admin/cluster/rebalanc, Calculates the current cluster imbalance}
 
 @RESTDESCRIPTION
-Calculates the current cluster imbalance and returns the result.
+Calculates the current cluster imbalance and returns the result. 
+It additionally shows the amount of ongoing and pending moveshard operations.
```

Comment:
please change to "pending move shard operations"

Reply:
```suggestion
It additionally shows the amount of ongoing and pending move shard operations.
```